package.json

```diff
@@ -25,7 +25,7 @@
     "build:es5": "BABEL_ENV=es5 babel lib -d dist/es5",
     "build:es-modules": "BABEL_ENV=es-modules babel lib -d dist/es-modules",
     "buildall": "npm run buildnode && npm run buildweb && npm run buildreactnative && npm run buildnativescript",
-    "buildnode": "webpack --config webpack/webpack.node.js --mode production",
+    "buildnode": "webpack --config webpack/webpack.node.js --mode development",
```

Comment:
Hi @vkalta,
Why did we change this node build from production to development?

Reply:
Hi @netrajpatel , thanks for catching this blunder. I have reverted it now.